test/05_convert.jl

```diff
@@ -22,6 +22,10 @@ module TestConvert
     convert(UInt8, v2)
     convert(UInt8, v3)
 
+    @test get(v1) == 1
```

Comment:
Use `===` as in other places: this allows testing for type equality, which is always a good thing in tests.